cutlass/group_mm.cu

```diff
@@ -430,14 +422,16 @@ void validateInputsGroupMm(
           b.scalar_type() == at::ScalarType::Half,
       "Expected BFloat16 or Half for Operand B.")
 
-  const int64_t m = a.sizes()[0];
-  const int64_t g = expert_offsets.sizes()[0];
-  int64_t prev_offset = 0;
-  for (int64_t i = 0; i < g; ++i) {
-    int64_t expert_offset = expert_offsets[i].item<int64_t>();
-    NVF_CHECK_LE(expert_offset, m);
-    NVF_CHECK_LE(prev_offset, expert_offset);
-    prev_offset = expert_offset;
+  if (at::cuda::currentStreamCaptureStatus() == at::cuda::CaptureStatus::None) {
+    const int64_t m = a.size(0);
+    const int64_t g = expert_offsets.size(0);
+    int64_t prev_offset = 0;
+    for (int64_t i = 0; i < g; ++i) {
+      const auto expert_offset = expert_offsets[i].item<int64_t>();
```

Comment:
This triggers a GPU-to-CPU copy so we skip this when the program runs in capture mode.